cmd/kubemark/hollow-node.go

```diff
@@ -181,52 +168,84 @@ func run(config *hollowNodeConfig) {
 	if len(config.TenantServers) == 0 {
 		klog.V(3).Infof("TenantServers is not set. Default to single tenant partition and clientConfig setting")
 		config.TenantServers = make([]string, 1)
-		config.TenantServers[0] = clientConfigs.GetConfig().Host
+		config.TenantServers[0] = config.KubeconfigPath
 	}
 	if config.ResourceServer == "" {
 		klog.V(3).Infof("Resource is not set. Default to clientConfig setting")
-		config.ResourceServer = clientConfigs.GetConfig().Host
+		config.ResourceServer = config.KubeconfigPath
```

Comment:
Argument type changes. Please make sure this works with scale out local cluster setup before merge to poc branch

Reply:
both scale up and scaleout were tested as described in the PR. actually the argument type is unlikely to change once it is determined, as string type for example in this case.